### ISSUE TITLE
Issue with add_nat_traversal when dumping server config

### ISSUE THREAD

**Issue by UrfinJuice11** [fictivekin/wireguard#12]
Hi, now using the following scheme to work with currnet wg config when adding a new Peer:

```def add_peer(name):
    if temp_conf_exist(TEMP_FILENAME):
        server = load_server(TEMP_FILENAME)
    else:
        server = create_server(NET, ADDRESS, PUBLIC, PRIVATE, ENDPOINT, PORT)
    peer = server.peer(name)
    public_key = peer.config().public_key.split(' = ')[1]
    private_key = peer.config().private_key.split(' = ')[1]
    address = peer.config().address.split('=')[1].strip()
    server.config().write()
    save_server(server, TEMP_FILENAME)
    if is_up(IFACE):
        server.service.restart()
    else:
        server.service.start()
    return public_key, private_key, address
```
1.  Create a new server object from scratch or from a pickle dump (**temp**) of previous configuration.
2. Create a new peer for this object.
3. Apply the configuration to create wg0.conf and wg0-peers.conf files
4. Save server_kwargs to a **temp** file.

So every time I perform any action on my working server I get the server object from **temp** file, perform the action and save the new object again. 

Everything worked well until I tried **add_nat_traversal** method. If I have no **temp** file and create the server as `server = Server()` and the use `server.add_nat_traversal` - everything is OK and I get proper PostUp settings in wg0.conf:

![image](https://user-images.githubusercontent.com/33019945/194286374-0b866bf3-e549-4c19-a99c-6ae7a43c2cc0.png)

But, adding one more user (after the cycle with **temp** file) I get the following:

![image](https://user-images.githubusercontent.com/33019945/194286598-6b0447e2-c03d-4ef1-a1e2-307143f374be.png)

So, my PostUp rules are interpreted as lists. Every new user adds a list layer:

![image](https://user-images.githubusercontent.com/33019945/194286832-658361c4-80c1-472d-890e-98536d767e3e.png)

Obviously, it is not interpreted properly by the wireguard. 

So, 
question 1:  Could you advise some walkaround for this situation?
question 2:  If I would like to hardcode the post_up rules to use them explicitly when creating the server: `server = Server(..., post_up=**xxx**)` what **xxx** format should be?

Regards

**Comment by jnhmcknight** [comment]
The problem with the `PostUp` rules getting nested on each subsequent import is actually a bug. I've corrected it in https://github.com/fictivekin/wireguard/commit/7ef9d169525ab6a166e40184f2db3a39a27efe1e (which also allows fixes #10). It can be released in the next few hours.

Also, wondering why you're using this:

```
 public_key = peer.config().public_key.split(' = ')[1]
 private_key = peer.config().private_key.split(' = ')[1]
 address = peer.config().address.split('=')[1].strip()
```

instead of directly using:

```
public_key = peer.public_key
private_key = peer.private_key
address = peer.address
```

...realizing that `peer.address` is actually a list, in case of the use of both IPv4/6, but `peer.ipv4` or `peer.ipv6` could be used instead.

**Comment by UrfinJuice11** [comment]
Thanks for quick reponse and work on the issue!

About my code: for that moment I didn't dig enouph into the code. Thank for help!
